bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,237 @@
+Shutdown turns the receiver off, which requires a manual PowerOn.
+
+There is no way to detect the "display status" of the receiver. 
+The binding detects Power-Off with the MR401B/MR201 by listening to UPnP events, but can't verify the status when started.
+You need to take care on the current status if you power on/off the receiver from scenes.
+Check the current status before sending the POWER button, because POWER is a toggle, not ON or OFF (see sample rules).
+
+### Discovery
+
+The auto discovery starts when the binding is loaded.
+UPnP will be used to discover receivers on the local network.
+Based on UPnP all necessary parameters are auto-discovered.
+The receiver needs to be powered on to get discovered.
+
+### Supported Things
+
+|Thing    |Type                                                           |
+|---------|-------------------------------------------------------------- |
+|receiver |A MagentaTV Receiver, the binding supports multiple receivers. |
+
+
+
+# Device setup
+
+Once the thing will be added from the Inbox in PaperUI you'll need your T-Online credentials to query the userID.
+Open the thing configuration and enter the credentials:
+
+|Field            |Description                                      |
+|---------------- |------------------------------------------------ |
+|Account Name     |Your T-Online user id, e.g. test7017@t-online.de |
+|Account Password |The password for your Telekom account.           |
+
+For security reasons the credentials will be automatically deleted from the thing configuration (replaced with '***' in the thing config) after the initial authentication process.
+The openHAB instance needs access to the Internet to perform that operation, which can be disabled afterwards.
+
+One the userID has been obtained from the Telekom portal the binding initiates the pairing with the receiver.
+This is an automated process.
+The thing changes to ONLINE state once the pairing result is received.
+Otherwise open PaperUI:Confuguration:Things and check for an error message.
+Using [Show Properties] you see more details and could verify if the discovery and pairing were completed successful.
+
+The binding uses the network settings in openHAB system configuration to determine the local ip address. The device can't be discovered when the openHAB system and receiver are not on the same network (ip/netmask).
+
+## Thing Configuration
+
+|Parameter       |Description                                                                                                     |
+|----------------|----------------------------------------------------------------------------------------------------------------|
+|udn             |UPnP Unique Device Name - a hex ID, which includes the 12 digit MAC address at the end (parsed by the binding)  |
+|modelId         |Type of Media Receiver: DMS_TPB for MR400 and MR200 ; MR401B for MR401B and MR201                               |
```

Comment:
Why is this parameter required? If it makes a difference, you should model it as different thing types, that's exactly what they are meant for.

Reply:
modelId is removed, was a left-over from initial implementation